serverless/email/events/event.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Should be updated to reflect `/webhook` and `/unsubscribe` events.  There should also be an event for `status` lambda where [`sam local invoke`](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-local-invoke.html) can be used.  To generate events, we should use [`sam local generate-event`](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-local-generate-event.html) for the given AWS service that will trigger the lambdas.